Assets/MRTK/Core/Utilities/MixedRealitySceneContent.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+#if UNITY_2017_2_OR_NEWER
+using System.Collections;
+using UnityEngine.XR;
+#else
+using UnityEngine.VR;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Use this script on GameObjects you wish to be aligned in certain ways depending on the application space type.
+    /// For example, if you want to place an object at the height of the user's head in a room scale application, check alignWithHeadHeight.
+    /// In a stationary scale application, this is equivalent to placing the object at a height of 0.
+    /// You can also specify specific locations to place the object based on space type.
+    /// 
+    /// This script runs once, on GameObject creation.
+    /// 
+    /// See https://developer.microsoft.com/en-us/windows/mixed-reality/coordinate_systems_in_unity for more information.
```

Comment:
```suggestion
    /// See https://docs.microsoft.com/windows/mixed-reality/coordinate_systems_in_unity for more information.
```
Also it seems that the linked page is pretty old - any chance that you could take a look and make updates to it in case it is outdated?

Reply:
Will do!